akka-actor-testkit-typed/src/test/scala/akka/actor/testkit/typed/scaladsl/ActorTestKitSpec.scala

```diff
@@ -21,6 +23,16 @@ class ActorTestKitSpec extends ScalaTestWithActorTestKit with AnyWordSpecLike wi
       system.name should ===("ActorTestKitSpec")
     }
 
+    "generate a default name & ActorSystemSetup from the provided actor" in {
```

Comment:
i think this test verifies that the provided actor system is used rather than anything related to ActorSystemSetup, rename?